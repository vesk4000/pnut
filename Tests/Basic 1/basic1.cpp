#include <bits/stdc++.h>
#include <Windows.h>
using namespace std;

int take_time() {
	int sum = 0;
	for(int i = 0; i < 1000000000; ++i) {
		sum += i;
		sum %= 10000;
	}
	return sum;
}

int arr[100000000];

int main() {
	int n;
	cin >> n;
	memset(arr, 1, sizeof(arr));
	for(int i = 0; i < n; ++i) {
		int a, b;
		cin >> a >> b;
		cout << a * b << endl;
		cout << arr[5] << endl;
		//cout << take_time() << endl;
	}
	return 0;
}