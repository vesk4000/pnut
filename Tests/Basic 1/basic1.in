5
3 4
1000 50
6 -1
-2 -3
20 0