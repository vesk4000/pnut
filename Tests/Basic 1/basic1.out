12
50000
-6
6
0
